actionview/lib/action_view/test_case.rb

```diff
@@ -234,6 +234,7 @@ def config
       end
 
       def render(options = {}, local_assigns = {}, &block)
+        @_rendered_view_content = nil
```

Comment:
@rafaelfranca Is this internal state reseting worth the maintenance cost when compared to removing the memoization instance variable entirely?

```diff
diff --git a/actionview/lib/action_view/test_case.rb b/actionview/lib/action_view/test_case.rb
index 97d1660200..006c6bf759 100644
--- a/actionview/lib/action_view/test_case.rb
+++ b/actionview/lib/action_view/test_case.rb
@@ -234,7 +234,6 @@ def config
       end
 
       def render(options = {}, local_assigns = {}, &block)
-        @_rendered_view_content = nil
         view.assign(view_assigns)
         @rendered << output = view.render(options, local_assigns, &block)
         output
@@ -292,7 +291,7 @@ def rendered_views
       #     assert_pattern { rendered.json => { title: "Hello, world" } }
       #   end
       def rendered
-        @_rendered_view_content ||= self.class.content_class.new(@rendered)
+        self.class.content_class.new(@rendered)
       end
 
       def _routes
@@ -389,7 +388,6 @@ def view
         :@_partials,
         :@passed,
         :@rendered,
-        :@_rendered_view_content,
         :@request,
         :@routes,
         :@tagged_logger,
```